src/umpire/ResourceManager.cpp

```diff
@@ -399,6 +405,40 @@ ResourceManager::move(void* ptr, Allocator allocator)
     return ptr;
   }
 
+#if defined(UMPIRE_ENABLE_NUMA)
+  {
+    // short circuit if allocator is a NUMA allocator
+    auto numa_alloc = std::dynamic_pointer_cast<strategy::NumaPolicy>(allocator.getAllocationStrategy());
+    if (numa_alloc) {
+      auto& op_registry = op::MemoryOperationRegistry::getInstance();
+
+      auto src_alloc_record = m_allocations.find(ptr);
+
+      const size_t size = src_alloc_record->m_size;
+      util::AllocationRecord dst_alloc_record;
+      dst_alloc_record.m_size = src_alloc_record->m_size;
+      dst_alloc_record.m_strategy = numa_alloc;
+
+      void *ret = nullptr;
+      if (size > 0) {
+        auto op = op_registry.find("RELOCATE",
```

Comment:
Can you register this as "MOVE" - I kept reading this as "REALLOCATE" and was thinking, "Why is it called reallocate?!